pp/bare_bones/bit_sequence.h

```diff
@@ -396,6 +396,22 @@ class PROMPP_ATTRIBUTE_PACKED CompactBitSequence : public CompactBitSequenceBase
     push_back_bits_u64(size_in_bytes << 3, val);
   }
 
+  PROMPP_ALWAYS_INLINE void push_back_bytes(const uint8_t* bytes, uint32_t count) noexcept {
+    reserve_enough_memory_if_needed(Bit::to_bits(count));
+
+    if (unfilled_bits_in_byte() == 0) [[unlikely]] {
+      std::memcpy(Base::memory_ + Base::size_in_bytes(), bytes, count);
+    } else {
+      for (uint32_t i = 0; i < count; ++i) {
```

Comment:
This code should be more efficient
```suggestion
      for (;count >= sizeof(uint64_t); count -= sizeof(uint64_t), bytes += sizeof(uint64_t) ) {
        push_back_u64(*reinterpret_cast<uint64_t*>(bytes));
      }
      
      for (;count >= sizeof(uint32_t); count -= sizeof(uint32_t), bytes += sizeof(uint32_t) ) {
        push_back_u32(*reinterpret_cast<uint64_t*>(bytes));
      }
      
      for (;count > 0; --count) {
        push_back_byte(bytes++);
      }
```